Decred Wallet/Features/Navigation Menu/MenuItem.swift

```diff
@@ -10,72 +10,44 @@ import UIKit
 
 enum MenuItem: String, CaseIterable {
     case overview = "Overview"
-    case history = "History"
-    case send = "Send"
-    case receive = "Receive"
-    case accounts = "Accounts"
-    case security = "Security"
-    case settings = "Settings"
-    case help = "Help"
+    case transactions = "Transactions"
+    case accounts = "Wallets"
+    case more = "More"
     
-    // Each menu item's VC is wrapped in a navigation controller to enable the display of a navigation bar on each page,
+    // Each menu item's VC is wrapped in a navigation controller to enable the display of a navigation bar on each page
     // and to allow each page perform VC navigations using `self.navigationController?.pushViewController`.
     var viewController: UIViewController {
         switch self {
         case .overview:
             return Storyboards.Overview.instantiateViewController(for: OverviewViewController.self).wrapInNavigationcontroller()
             
-        case .history:
+        case .transactions:
             return TransactionHistoryViewController(nibName: "TransactionHistoryViewController", bundle: nil).wrapInNavigationcontroller()
-            
-        case .send:
-            return SendViewController.instance
-            
-        case .receive:
-            return Storyboards.Main.instantiateViewController(for: ReceiveViewController.self).wrapInNavigationcontroller()
-            
+        
         case .accounts:
             return Storyboards.Main.instantiateViewController(for: AccountViewController.self).wrapInNavigationcontroller()
-            
-        case .security:
-            return Storyboards.SecurityMenu.instantiateViewController(for: SecurityMenuViewController.self).wrapInNavigationcontroller()
-            
-        case .settings:
-            return SettingsController.instantiate().wrapInNavigationcontroller()
-            
-        case .help:
-            return Storyboards.Main.instantiateViewController(for: HelpViewController.self).wrapInNavigationcontroller()
+        
+        case .more:
+            return Storyboards.Overview.instantiateViewController(for: OverviewViewController.self).wrapInNavigationcontroller() // Change this when settings view controller is ready
```

Comment:
I'm guessing this is temporary, should add a todo comment here. Also, as a temporary measure, instead of showing the overview page on more tap, show the settings page.